app/main.py

```diff
@@ -172,21 +251,25 @@ async def program():
         raise HTTPException(status_code=NOT_FOUND, detail=str(e))
 
 
-@app.get("/projects/{program}")
+@app.post("/project")
```

Comment:
This is because the URL style I prefer would be "/{programs}/project", which we can clearly see its level. As you told me last time, you prefer this style ("/projects/{program}"). I think the level will not be clear, so I decided to make the URL style like other routes. To be more simple.